api/src/test/java/org/openmrs/hl7/HL7ServiceTest.java

```diff
@@ -736,6 +737,7 @@ public void resolveLocationId_shouldReturnNullIfLocationIdAndNameAreIncorrect()
 	@Test
 	public void resolveUserId_shouldReturnNullForAmbiguousUsersUsingFirstAndLastNameGivenUserIDIsNull() throws HL7Exception {
 		HL7Service hl7service = Context.getHL7Service();
+		executeDataSet(XML_FILENAME);
```

Comment:
What is this dataset for? In other words, what happens if we do not have it?

Reply:
I added some User to replace the Daemon User which is filter out as an effect of my changes. Removing this makes the test to fail. @dkayiwa